tests/strings/test_manacher.py

```diff
@@ -0,0 +1,15 @@
+from strings.manacher import palindromic_string
+
+
+def test_manacher_find_palindromic_string():
+    """Testing if a string is palindromic or not"""
+    n = "abba"
+    assert palindromic_string(n) == "abba"
+
+
+def test_manacher_return_output_from_center():
+    """Testing if a string is not palindromic then return the center character"""
+    n = "qwerty"
+    assert palindromic_string(n) == "w"
```

Comment:
Remove this assert. The third assert tests that it is in `n`, but this assertion will fail if it isn't "w", meaning the third assertion has no effect.